library/RAVE/src/RAVE/audio/IO/IO_manager.py

```diff
@@ -0,0 +1,45 @@
+from pyodas.io import WavSource, MicSource, PlaybackSink, WavSink
+
+SOURCE_TYPES = {
+    "source_wav": WavSource,
+    "source_mic": MicSource,
+}
+
+SINK_TYPES = {
+    "sink_wav": PlaybackSink,
+    "sink_playback": WavSink,
+}
+
+
+class IOManager:

```

Comment:
We should maybe look into being able to have multiple sinks (for example, you might want to save the unprocessed and processed signal and also playback the audio signal).

Reply:
Good idea, will add a dict.